tests/ops/api/v1/endpoints/test_config_endpoints.py

```diff
@@ -720,6 +729,16 @@ def test_put_application_config_updates_cors_domains_in_middleware(
         response = api_client.options(url, headers=headers)
         assert response.status_code == 200
 
+        # but ensure that an unrelated origin is still not allowed
+        headers = {
+            **auth_header,
+            "Origin": "http://fakesite.com",
```

Comment:
SUPER-nit:
```suggestion
            "Origin": "https://fakesite.com",
```

I know it's a weird comment, but it's the right muscle to assume `https` in all our examples and tests and everywhere as an expected default.

Reply:
fair enough!

Reply:
addressed in https://github.com/ethyca/fides/pull/4584/commits/047c7c48497daec1239b1d7db6bdc7bd4f3e1b29